MangoAPI.Presentation/Controllers/OpenSslKeyExchangeController.cs

```diff
@@ -65,7 +102,18 @@ public async Task<IActionResult> OpenSslGetDiffieHellmanParameter(CancellationTo
         return await RequestAsync(command, cancellationToken);
     }
 
+    /// <summary>
+    /// Get OpenSSL key exchange requests.
+    /// </summary>
+    /// <param name="cancellationToken">Cancellation token instance.</param>
+    /// <returns></returns>
     [HttpGet]
+    [SwaggerOperation(
+        Summary="Get key exchange requests.",
+        Description="Get OpenSSL exchange requests.")]
+    [ProducesResponseType(typeof(BadRequest), StatusCodes.Status400BadRequest)]
```

Comment:
Please check the type of responses at the handlers, the responses are `ErrorResponse` (409) or `T` (200) of `Result<T>`